client/scripts/views/components/new_proposal_button.ts

```diff
@@ -120,7 +120,12 @@ export const getNewProposalMenu = (candidates?: Array<[SubstrateAccount, number]
     }),
     showSnapshotOptions && m(MenuItem, {
       onclick: (e) => {
-        navigateToSubpage(`/new/snapshot/${app.chain.meta.chain.snapshot}`);
+        const snapshotSpaces = app.chain.meta.chain.snapshot;
```

Comment:
I feel like we don't need to pull out the constant here, or we use the constant in line 127 instead of `app.chain.meta.chain.snapshot` or drop the long reference in line 124 (it's not *too* long)